src/ResourceManager/DataMigration/Commands.DataMigration/Models/MongoDb/MongoDbCollectionSetting.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.Azure.Commands.DataMigration.Models.MongoDb
```

Comment:
@haitsongmsft please make sure to add the Microsoft license header to all `.cs` files that are missing it

Reply:
updated